.env.docker

```diff
@@ -3,4 +3,6 @@ PUBLIC_POCKETBASE_URL=http://grimoire-pocketbase:80
 # RECOMMENDED: Change this to your email
 ROOT_ADMIN_EMAIL=admin@grimoire.localhost
 # RECOMMENDED: Use secure password. Can be later changed in the PocketBase admin panel
-ROOT_ADMIN_PASSWORD=changeme
\ No newline at end of file
+ROOT_ADMIN_PASSWORD=changeme
+# Set this to your domain name
+ORIGIN=http://localhost:5137
```

Comment:
Should this be ORIGIN_URL, base on the additions made to the Dockerfile?

Reply:
Yes, but to simplify orchestration of environment variables, I left it now as just `ORIGIN`.